### ISSUE TITLE
please help me，why are the images of the validation set used during the training process?

### ISSUE THREAD

**Issue by tianhaoliuzhu** [bubbliiiing/yolov4-pytorch#372]
![123](https://github.com/bubbliiiing/yolov4-pytorch/assets/100944374/810a941a-2139-4301-8ca1-ad3d240115dc)

**Comment by MingAze** [comment]
ZjZkJ送 Q 币IdMaFhttp://service.qq.com/ptLogin.shtml?url=http://lai2z.spaces.live.com/w4M2P
